content/how-to/rackspace-email/cloud-office-mfa-app-guide/index.md

```diff
@@ -0,0 +1,225 @@
+---
+permalink: cloud-office-mfa-app-guide/
+audit_date: '2020-07-27'
+title: Multifactor Authentication App Setup
```

Comment:
Should we make this be something more like "Set up an app for multi-factor authentication"?

Reply:
ooooh, good idea!  Doh!

Reply:
fixed, @stephamon